migrations/20201015005801-update-user-roles.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
```

Comment:
we also need to drop the `accessLevel` enum from the `users` table as well - [there's an example of it here](https://github.com/uwblueprint/paramedics-web/blob/a47ee4e7c19d4fc575bd2c870d7a896f5888c531/migrations/20200128014549-create-user.js#L44)

(and don't forget to put it back in during the down migration)

(also you'll need to drop it last within the up migration and add it first within the down migration)

Reply:
Agree with this! You can check if you are deleting enums properly with [this query](https://stackoverflow.com/questions/9540681/list-postgres-enum-type).

Reply:
ah ok I forgot about that! will add that in